source/_integrations/zwave.markdown

```diff
@@ -20,6 +20,12 @@ ha_codeowners:
 ha_domain: zwave
 ---
 
+<p class='note warning'>
+This integration is deprecated. We recommend using <a href="/integrations/zwave_js">the Z-Wave JS integration</a>.
+<br><br>
+The Z-Wave integration will no longer receive any updates. It will not be removed unless it becomes incompatible with a future version of Python.
+</p>
```

Comment:
```suggestion
<p class='note warning'>

This integration is deprecated. We recommend using [the Z-Wave JS integration](/integrations/zwave_js).

The Z-Wave integration will no longer receive any updates. It will not be removed unless it becomes incompatible with a future version of Python.

</p>
```